site/src/components/RichParameterInput/RichParameterInput.tsx

```diff
@@ -189,16 +209,33 @@ export const RichParameterInput: FC<RichParameterInputProps> = ({
     >
       <ParameterLabel parameter={parameter} />
       <div css={{ display: "flex", flexDirection: "column" }}>
-        <RichParameterField {...fieldProps} size={size} parameter={parameter} />
-        {autofillSource && autofillSource !== "active_build" && (
+        <RichParameterField
+          {...fieldProps}
+          onChange={onChange}
+          size={size}
+          parameter={parameter}
+        />
+        {!parameter.ephemeral &&
```

Comment:
Why do we check if it is not ephemeral?

Reply:
`ephermal` basically means "should not be reused"